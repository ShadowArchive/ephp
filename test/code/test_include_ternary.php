<?php
WP_DEBUG ? include(WP_CONTENT_DIR) : @include(WP_CONTENT_DIR);
