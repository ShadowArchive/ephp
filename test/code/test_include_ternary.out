
Notice: Use of undefined constant WP_DEBUG - assumed 'WP_DEBUG' in {{CWD}}/test/code/test_include_ternary.php on line 2

Notice: Use of undefined constant WP_CONTENT_DIR - assumed 'WP_CONTENT_DIR' in {{CWD}}/test/code/test_include_ternary.php on line 2

Warning: include(WP_CONTENT_DIR): failed to open stream: No such file or directory in {{CWD}}/test/code/test_include_ternary.php on line 2

Warning: include(): Failed opening 'WP_CONTENT_DIR' for inclusion (include_path='.:') in {{CWD}}/test/code/test_include_ternary.php on line 2
